src/vma/proto/dst_entry_udp_mc.cpp

```diff
@@ -81,6 +81,24 @@ bool dst_entry_udp_mc::conf_l2_hdr_and_snd_wqe_ib()
 	return ret_val;
 }
 
+void dst_entry_udp_mc::set_src_addr()
+{
+	m_pkt_src_ip = INADDR_ANY;
+	
+	if (m_bound_ip) {
+		m_pkt_src_ip = m_bound_ip;
```

Comment:
If I execute bind() after sending the first packet:
Source IP for MCAST packet will be different than source IP for Unicast packet.
If correct - please justify this difference